### ISSUE TITLE
Error met while evaluating models: 'module' object has no attribute 'Resize'

### ISSUE THREAD

**Issue by ercanburak** [KaiyangZhou/deep-person-reid#11]
Hello @KaiyangZhou, 

Thank you for this repo. I get an error while trying to evaluate some of the pretrained models. Here are the details:

This is my first time trying this repo. In a clean conda python2.7 environment, I installed PyTorch and torchvision, and the installations seem fine. I prepared the Market1501 dataset and downloaded some pretrained model weights as described in the readme. Then I tried to evaluate the ResNet50M xent+htri pretrained model for Market1501. I used the command as: 

`python train_img_model_xent_htri.py -d market1501 -a resnet50m --evaluate --resume saved-models/resnet50m_xent_htri_market1501.pth.tar --save-dir log/resnet50m-xenthtri-market1501 --test-batch 32`

I got the following error logs:

```
==========
Args:Namespace(arch='resnet50m', cuhk03_classic_split=False, cuhk03_labeled=False, dataset='market1501', eval_step=-1, evaluate=True, gamma=0.1, gpu_devices='0', height=256, htri_only=False, lr=0.0003, margin=0.3, max_epoch=180, num_instances=4, optim='adam', print_freq=10, resume='saved-models/resnet50m_xent_htri_market1501.pth.tar', root='data', save_dir='log/resnet50m-xenthtri-market1501', seed=1, split_id=0, start_epoch=0, stepsize=60, test_batch=32, train_batch=32, use_cpu=False, use_metric_cuhk03=False, weight_decay=0.0005, width=128, workers=4)
==========
Currently using GPU 0
Initializing dataset market1501
=> Market1501 loaded
Dataset statistics:
  ------------------------------
  subset   | # ids | # images
  ------------------------------
  train    |   751 |    12936
  query    |   750 |     3368
  gallery  |   751 |    15913
  ------------------------------
  total    |  1501 |    32217
  ------------------------------
Traceback (most recent call last):
  File "train_img_model_xent_htri.py", line 295, in <module>
    main()
  File "train_img_model_xent_htri.py", line 115, in main
    T.Resize((args.height, args.width)),
AttributeError: 'module' object has no attribute 'Resize'
```

I thought this might be a problem with torchvision, but the latest version of it (0.2.1) is installed. What am I missing here? Can you give some guidance?

**Comment by KaiyangZhou** [comment]
Are you using the latest master branch? What pytorch version you are using?

**Comment by ercanburak** [comment]
Yes, I am using the latest master branch. Pytorch is 0.4.0.

**Comment by KaiyangZhou** [comment]
This is strange. I met such error only when using old pytorch. It looks like that the import in [transforms.py](https://github.com/KaiyangZhou/deep-person-reid/blob/master/transforms.py#L3) was not successful so `T` has no method `Resize`. Could you double check the environment set up is correct?

You can also quickly try creating a script (say trans.py) with only this line `from torchvision.transforms import *` and do `import trans` in another script and `tr = trans.Resize((10, 10))`. See if you get an error with this.

**Comment by ercanburak** [comment]
I resolved the issue, it was due to the torchvision installation:

1) When I install torchvision by conda, the error occurs, resize not found.
2) When I install torchvision by pip, the error occurs, resize not found.
3) When I install torchvision from source (and add its path to PYTHONPATH), I do not see this error.

I do not know why installing by conda or pip results in this situation. However, I see in [another repo](https://github.com/layumi/Person_reID_baseline_pytorch) that it is explicitly stated as "Install Torchvision from the source". If this is the case, it might be a good idea to add something like that to the readme.
